ledger-crypto-rs/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "ledger-crypto"
+description = "Library to integrate with the Ledger Crypto app"
+version = "0.1.0"
+license = "Apache-2.0"
+authors = ["Zondax GmbH <info@zondax.ch>"]
+homepage = "https://github.com/Zondax/ledger-crypto-rs"
+repository = "https://github.com/Zondax/ledger-crypto-rs"
+readme = "README.md"
+categories  = ["authentication", "cryptography"]
+keywords = ["ledger", "nano", "apdu", "crypto"]
+edition     = "2018"
+autobenches = false
+
+[badges]
+circle-ci = { repository = "zondax/ledger-crypto-rs" }
+
+[lib]
+name = "ledger_crypto"
+
+[dependencies]
+byteorder = "1.3.4"
+lazy_static = "1.4.0"
+serde = { version = "1.0", features = ["derive"] }
+thiserror = "1.0.20"
+log="0.4.8"
+
+zx-bip44 = { git = "https://github.com/Zondax/blockchain-tools-rs" }
+ledger-zondax-generic = { git = "https://github.com/Zondax/ledger-rs", branch="master" }
+ledger-transport = { git = "https://github.com/Zondax/ledger-rs", branch="master" }
+ledger-transport-hid = { git = "https://github.com/Zondax/ledger-rs", branch="master" }
+hex = "0.4"
+
+[dev-dependencies]
+futures = "0.3.5"
+matches = "0.1.8"
+serial_test = "0.4.0"
+env_logger = "0.7.1"
+secp256k1 = "0.17.2"
+blake3 = "0.3.5"
+tokio = { version = "0.2", features = ["full"] }
+tokio-test = "0.2.1"
```

Comment:
are all these dependencies necessary ? for `secp256k1`, could it use the same version as the rest of the repository?

Reply:
I'll remove them, because I removed the integration tests in ledger-crypto-rs,  no need anymore.